cmd/networking/certmanager/main.go

```diff
@@ -24,6 +24,6 @@ import (
 )
 
 func main() {
-	sharedmain.Main("controller-certificate-cert-manager",
+	sharedmain.Main("certcontroller",
```

Comment:
```suggestion
	sharedmain.Main("cert_controller",
```
same below?

Reply:
See https://prometheus.io/docs/practices/naming/#metric-names

> A metric name should have a (single-word) application prefix relevant to the domain the metric belongs to. The prefix is sometimes referred to as namespace by client libraries. For metrics specific to an application, the prefix is usually the application name itself.

Reply:
That's supper annoying

Reply:
Yes.

Reply:
Probably worth extracting into a constant with a link to that doc in the doc comment to avoid accidental changes.